lib/Item.php

```diff
@@ -71,7 +71,7 @@ class Item implements IScannable {
 	 * @throws NotFoundException
 	 * @throws QueryException
 	 */
-	public function __construct(IL10N $l10n, View $view, string $path, int $id = null, string $etag = null) {
+	public function __construct(IL10N $l10n, View $view, string $path, ?int $id, ?string $etag) {
```

Comment:
```suggestion
	public function __construct(IL10N $l10n, View $view, string $path, ?int $id = null, ?string $etag = null) {
```

We should be able to still specify that `null` is the default value. Then callers are not forced to pass `null` explicitly.